packages/webpack/src/presets/base.ts

```diff
@@ -40,7 +42,7 @@ function baseConfig (ctx: WebpackConfigContext) {
     mode: ctx.isDev ? 'development' : 'production',
     cache: getCache(ctx),
     output: getOutput(ctx),
-    stats: 'none',
+    stats: statsMap[ctx.nuxt.options.logLevel],
```

Comment:
As best practice, we might map to a default level `statMaps[logLevel] || statMaps[info]` in case of receiving an unexpected level.